policy-controller/grpc/src/lib.rs

```diff
@@ -320,5 +342,198 @@ fn to_authz(
         networks,
         labels,
         authentication: Some(authn),
+        metadata: None, // TODO fill
+    }
+}
+
+fn to_http_route(
+    name: &String,
```

Comment:
If we really want a reference, `&str` is better than `&String` -- we could have a `&'static str` or an `Arc<str`> or etc, and we'd just want to use a reference to the `str` rather than require a reference to an owned `String`...

But `impl ToString` is probably better yet in this case.